From f20640f677a24e30faa3fe7e35fb5dae4206ff63 Mon Sep 17 00:00:00 2001
From: Ermine Jose <quic_erminejo@quicinc.com>
Date: Tue, 9 Jan 2024 15:45:08 +0530
Subject: [PATCH] include: btrfs_tree.h: enable the BTRFS_EXTENT_REF_V0_KEY
 macro

The macro BTRFS_EXTENT_REF_V0_KEY has dependency on strace
package and is still valid with OpenWRT in kernel-headers,
hence uncomment the macros definition.

Signed-off-by: Ermine Jose <quic_erminejo@quicinc.com>
Change-Id: Ib41b1462c84dfedd66078ac1e16da159ae2d5123
---
 include/uapi/linux/btrfs_tree.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/uapi/linux/btrfs_tree.h b/include/uapi/linux/btrfs_tree.h
index fc3c32186d7e..a4cabade3b85 100644
--- a/include/uapi/linux/btrfs_tree.h
+++ b/include/uapi/linux/btrfs_tree.h
@@ -222,9 +222,9 @@
 
 /*
  * Obsolete key. Defintion removed in 6.6, value may be reused in the future.
- *
- * #define BTRFS_EXTENT_REF_V0_KEY	180
+ * As OpenWRT package, strace depends on this macro, temporarily enable it.
  */
+#define BTRFS_EXTENT_REF_V0_KEY		180
 
 #define BTRFS_SHARED_BLOCK_REF_KEY	182
 
-- 
2.34.1

